src/System.Management.Automation/engine/ManpageInfo.cs

```diff
@@ -0,0 +1,122 @@
+#if UNIX
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
```

Comment:
```suggestion

#if UNIX
#nullable enable

using System.Collections.Generic;
```

Reply:
Did not add the `#nullable enable` because it caused "_error CS8603: Possible null reference return_" for `OutputType` setting to `null` (not sure why): https://github.com/davidBar-On/PowerShell/blob/fa5ff7053df6c80242fcf58b0eb2587e964b777f/src/System.Management.Automation/engine/ManpageInfo.cs#L90-L93

Reply:
Do return nullable?
```public override ReadOnlyCollection<PSTypeName>? OutputType```

Reply:
Changed and added the `#nullable enable`.